src/config.rs

```diff
@@ -99,6 +99,16 @@ impl Environment {
         self.get_build_var("TARGET")
             .or_else(|| std::env::var("CARGO_BUILD_TARGET").ok())
     }
+
+    fn doctests(&self) -> Option<bool> {
+        env::var("CROSS_UNSTABLE_ENABLE_DOCTESTS")
+            .map(|s| bool_from_envvar(&s))
+            .ok()
+    }
+
+    fn custom_toolchain(&self) -> bool {
```

Comment:
I think it's better we don't process all our config options for environment variables in `lib` itself, since variables like this are part of the configuration.